GbengaOshinaga/more-recipes
# Feature/state maintenance

#### What does this PR do?
It maintains state of recipes to know which recipe has been voted or favourited by user
#### Description of Task to be completed?
N/A
#### How should this be manually tested?
Start app:
```npm run startServer```
```npm run startReact```
Navigate to:
```localhost:3000/catalog```
Vote or favourite, then refresh page to see the color not change
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
N/A
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A
